app/src/main/java/space/taran/arknavigator/mvp/presenter/adapter/tagsselector/TagsSelectorPresenter.kt

```diff
@@ -248,6 +258,74 @@ class TagsSelectorPresenter(
         }
     }
 
+    private fun sort(
+        tagsOfSelectedResources: List<TagItem>,
+        tagsOfUnselectedResources: List<TagItem>,
+        tagItemsByResources: Map<ResourceId, Set<TagItem>>
+    ) {
+        when (sorting) {
+            TagsSorting.POPULARITY -> sortByPopularity(
+                tagsOfSelectedResources,
+                tagsOfUnselectedResources,
+                tagItemsByResources
+            )
+            TagsSorting.LAST_USED -> sortByLastUsed()
```

Comment:
I think we should also add `tag-labeled-n` but I can't find proper name for this metric :)
Maybe "activity" or "frequency"?

Reply:
There is already `Popularity` here.
Although it is manually calculated each time a tag is selected and `tag-labeled-n` metric is not used

Reply:
Actually "popularity" differs from `tag-labeled-n`. "Popularity" is current amount of resource labeled by a tag. "Activity" would be metric of "how often the tag is used", both labeling and unlabeling events increase this metric. It would put temporary tags like "todo" into the top of tag selector.